autonomi/src/wallet/error.rs

```diff
@@ -1,3 +1,4 @@
+#[cfg(feature = "transfers")]
```

Comment:
This is not needed as the feature gate is already in `lib.rs` on the `mod wallet`.